jans-cedarling/cedarling_opa/demo/terraform/README.md

```
@@ -0,0 +1,344 @@
+
+To stop and remove the containers:
+
+```bash
+docker compose down
+```
+
+---
+
+## Prerequisites
+
+### Docker path (recommended)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)